payment-request/payment-response/helpers.js

```diff
@@ -75,5 +75,24 @@ async function runManualTest(button, options, expected = {}, id = undefined) {
         `Expected response ${attribute} attribute to be ${value}`
       );
     }
+    // Special handling for option.requestShipping
+    assert_equals(
```

Comment:
Maybe a spec issue here, for the case that giving shippingOptions when options.requestShipping is false.
Do we have CR for this?